### PR TITLE
docs: add ng-India to events list

### PR DESCRIPTION
**Pull request by debugmodedotnet**
docs(docs-infra): add ng-India to events list on angular.io/events

added "ng-India" to the events list on angular.io/events

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Review by gkalpak — COMMENTED** [review]
Can you, please, change the commit message to `docs: add ng-India to events list`?

**Comment by debugmodedotnet** [comment]
Hi @StephenFluin, can you please review this? thanks.

**Review by StephenFluin — CHANGES_REQUESTED** [review]
Happy to add the 2019 event, but let's leave out the 2018 dates. We're going to be cleaning up "past events" soon.

**Review by debugmodedotnet — COMMENTED** [review]
removed past event details

**Review by debugmodedotnet — COMMENTED** [review]
docs: add ng-India to events list

**Comment by debugmodedotnet** [comment]
Hi @StephenFluin, I've removed the past event details, please review this. thanks!

File: aio/content/marketing/events.html [hunk 1]
```diff
@@ -79,7 +85,7 @@ <h1 class="banner-headline no-toc no-anchor">Events</h1>
       <th><a href="https://ngvikings.org/" title="ngVikings">ngVikings</a></th>
       <td>Helsinki, Finland</td>
       <td>March 1-2, 2018</td>
-    </tr>
+    </tr>
```
Comment (StephenFluin) [inline aio/content/marketing/events.html]:
```suggestion
    </tr>
```

**Review by StephenFluin — CHANGES_REQUESTED** [review]
One tiny change and then you'll need to squash your commits.
https://twitter.com/stephenfluin/status/1009904095073718275?lang=en

File: aio/content/marketing/events.html (hunk 1 above)
Comment (debugmodedotnet) [reply aio/content/marketing/events.html]: thanks @StephenFluin : I've made the necessary changes, please see if it looks good now, thanks.

**Review by StephenFluin — APPROVED** [review]

**Comment by debugmodedotnet** [comment]
thanks @StephenFluin, not sure why the tests failed!

**Pull request closed without merging**